.github/workflows/npm.yml

```diff
@@ -0,0 +1,33 @@
+name: Build and publish npm package
+
+on:
+  workflow_dispatch:
```

Comment:
I made this a manual workflow for now, because I wasn't sure exactly what we'd want the process for bumping the version in package.json to look like. It seems like maybe that version number should only be bumped when the loam-cli's version is released. Curious about what others think!

Reply:
Yeah in the past I found it easiest to keep the versions in sync. I think for now we can go with manual just to start, but ideally it would be automatic.